Assets/HoloToolkit/UX/Scripts/Keyboard.cs

```diff
@@ -218,10 +234,48 @@ protected override void Awake()
             RectTransform rect = GetComponent<RectTransform>();
             m_ObjectBounds = new Vector3(canvasBounds.size.x * rect.localScale.x, canvasBounds.size.y * rect.localScale.y, canvasBounds.size.z * rect.localScale.z);
 
+            // Actually find microphone key in the keyboard
+            var dicationButton = RecursiveFindChild(gameObject.transform, "Dictation");
```

Comment:
Why not assign this object in the inspector, so we don't have to look it up each time?

Reply:
Lookup is only used at Awake, so once in every time it starts